leku/src/main/java/com/schibstedspain/leku/geocoder/GeocoderPresenter.java

```diff
@@ -56,6 +59,7 @@ public void getFromLocationName(String query) {
         .subscribeOn(Schedulers.newThread())
         .observeOn(scheduler)
         .retry(RETRY_COUNT)
+        .onErrorResumeNext(apiInteractor.getFromLocationName(query))
```

Comment:
I was wondering it myself, but there are "missing" layers. So I've decided to make the interface useful. It can be moved to the interactor, but I think that I won't call it interactor, but datasource :)